src/sections/ContactUs/ContactUs.jsx

```diff
@@ -24,7 +24,7 @@ function useEmail(formId){
     e.preventDefault();
     const form = document.getElementById(formId)
     const formData = new FormData(form)
-    const url = 'https://formsubmit.co/72bebc68699d2efa6a8744eabeb45b2d';
```

Comment:
TODO: ask the person who this email belongs to, to give us the encoded string to be used in the code. this is important to prevent his email from being publicly exposed

Reply:
Will do before merging